lib/src/github/api.rs

```diff
@@ -960,4 +961,14 @@ impl Session for GithubSession {
             .error_for_status()?;
         Ok(())
     }
+
+    async fn get_team_members(&self, org: &str, team: &str) -> Result<Vec<User>> {
+        let users: Result<Vec<User>> = self
+            .client
+            .get(&format!("orgs/{}/teams/{}", org, team))
+            .await
+            .map_err(|e| format_err!("Error looking up team members {}/{}: {}", org, team, e));
+
+        Ok(users?)
```

Comment:
can simplify down to this:

```suggestion
        self.client
            .get(&format!("orgs/{}/teams/{}", org, team))
            .await
            .map_err(|e| format_err!("Error looking up team members {}/{}: {}", org, team, e))
```


The reason being that `Ok(users?)` basically unwraps the `Result<T>` into a `T` with the `?` operator, and then puts it back into a `Result<T>` with `Ok()`.


        Ok(users?)